crates/nu-protocol/src/value/mod.rs

```diff
@@ -3984,6 +3996,52 @@ mod tests {
     use super::{Record, Value};
     use crate::record;
 
+    mod error {
```

Comment:
I'd rather these this test be with the other `ls` tests in crates/nu-command/tests/commands/ls.rs